tests/spread/core24/linters-pack/library-gnome/snap/snapcraft.yaml

```diff
@@ -74,7 +64,9 @@ parts:
       - -Ddisable-introspection=true
     organize:
       snap/gnome-calculator/current/usr: usr
-    build-snaps: [ gnome-42-2204-sdk/latest/edge ]
+    build-snaps:
+      - gnome-46-2404-sdk/latest/edge
+      - gnome-46-2404/latest/edge
```

Comment:
is adding the content snap here for linting purposes?

Reply:
yes, but no, we require it as it is defined in `default-provider` so we try to download the snap for it for linting purposes, this should go away once the snap is in stable